core/camera/src/androidTest/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCaseTest.kt

```diff
@@ -183,7 +190,7 @@ class CameraXCameraUseCaseTest {
         val onRecorded = CompletableDeferred<Unit>()
         val onRecordStatus = CompletableDeferred<Unit>()
         var statusCount = 0
-        startVideoRecording {
+        startVideoRecording(null, false) {
```

Comment:
Is this change related to this PR?

Reply:
No but somehow the two parameters are missing and causes the compilation to fail. Somehow this got merged